packages/buidler-core/test/internal/buidler-evm/provider/modules/eth.ts

```diff
@@ -797,17 +797,615 @@ describe("Eth module", function() {
 
       assert.lengthOf(blockHashes, 3);
     });
+
+    it("should return reorganized block", async function() {
+      const filterId = await this.provider.send("eth_newBlockFilter", []);
+
+      assert.lengthOf(
+        await this.provider.send("eth_getFilterChanges", [filterId]),
+        1
+      );
+
+      await this.provider.send("evm_mine", []);
+      const block1 = await this.provider.send("eth_getBlockByNumber", [
+        await this.provider.send("eth_blockNumber"),
+        false
+      ]);
+
+      const snapshotId: string = await this.provider.send("evm_snapshot", []);
+      await this.provider.send("evm_revert", [snapshotId]);
```

Comment:
I don't get this test, because you are reverting without any new block being mined.

Reply:
🤦‍♂